drivers/hmis/app/graphql/types/application/user.rb

```diff
@@ -30,6 +32,19 @@ class Application::User < Types::BaseObject
     field :date_created, GraphQL::Types::ISO8601DateTime, null: false
     field :date_deleted, GraphQL::Types::ISO8601DateTime, null: true
 
+    audit_history_field(
+      filter_args: { type_name: 'UserAuditEvent' },
```

Comment:
```suggestion
      filter_args: { type_name: 'UserAuditEvent', omit: [:user] },
```

This will omit the `user` option from the filter options. It doesn't make much sense to filter by User since everything is related to the same user.

Reply:
after implementing this change I see an error:
```
undefined method `user' for #<#<Class:0x0000ffff6b4cfe80>:0x0000ffff7c651d60 @context=#<Query::Context ...>, @ruby_style_hash={}, @arguments=#<GraphQL::Execution::Interpreter::Arguments @keyword_arguments={}>> scope = scope.where(v_t[:user_id].in(filters.user).or(v_t[:whodunnit].in(filters.user)).or(v_t[:true_user_id].in(filters.user))) if filters&.user&.present? ^^^^^^
```
I'll troubleshoot but if I haven't figured it out tomorrow maybe we can pair on this as well